app/src/main/java/com/example/csc207courseproject/ui/report/ReportFragment.java

```diff
@@ -1,35 +1,47 @@
 package com.example.csc207courseproject.ui.report;
 
```

Comment:
Make sure you remove the ViewModelProvider from the framework and use our view model class